app/controllers/SubmissionsController.scala

```diff
@@ -40,44 +37,35 @@ class SubmissionsController @Inject() (
   verifyEmail: VerifiedEmailAction,
   customsDeclareExportsConnector: CustomsDeclareExportsConnector,
   mcc: MessagesControllerComponents,
-  submissionsPage: submissions,
   submittedDeclarationPage: submitted_declaration_page
-)(implicit ec: ExecutionContext, paginationConfig: PaginationConfig)
+)(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport {
 
-  def amend(rejectedId: String): Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
+  private val actions = authenticate andThen verifyEmail
+
+  def amend(rejectedId: String): Action[AnyContent] = actions.async { implicit request =>
```

Comment:
It was not removed. `authenticate andThen verifyEmail` is now represented by `actions`.

Reply:
Ok, I missed this as skim reading it looks awfully like the default `Action.async` method. Maybe if you rename the `actions` value to something else like `authAndEmailActions` it would be more obvious :slightly_smiling_face: